virt/monitoring/virt-monitoring-overview.adoc

```diff
@@ -28,5 +28,8 @@ Configure readiness, liveness, and guest agent ping probes and a watchdog for VM
 xref:../../virt/monitoring/virt-runbooks.adoc#virt-runbooks[Runbooks]::
 Diagnose and resolve issues that trigger {VirtProductName} xref:../../monitoring/managing-alerts.adoc#managing-alerts[alerts] in the {product-title} web console.
 
+Monitoring HCO health status::
+View the overall health of the Hyperconverged Cluster Operator (HCO) in the web console overview on the OCP overview dashboard in the operators' status card. The health is calculated using active alerts and HCO conditions and will show a *Healthy*, *Warning*, or *Critical* status.
```

Comment:
Are these the correct status? `*Healthy*, *Warning*, or *Critical*` can you please double check? From what I am observing, I see "Degraded" when health status is supposed to be critical.

Reply:
Tooltip shows warning, error etc.

Reply:
@dbasunag I'm seeing "Critical" and "Warning" in the test 4.14 test UI (Home > Overview > Status Card)
![image](https://github.com/openshift/openshift-docs/assets/119876664/b82542c4-9655-4604-8756-e7ff0f30a786)

Reply:
@danielclowers In the screenshot `0 Critical` indicates the number of critical alerts in firing state and similarly `4 Warning` indicates the number of alerts with Warning label (operator_health_impact). They don't stand  for the health metrics. I believe the "Degraded" one is currently communicating the health status. @sradco please correct me if my assumption is not correct.

Reply:
@dbasunag my mistake - i thought we were looking at the individual alerts and not the aggregate where it does indeed show "degraded". i've changed the vocabulary a bit on my most recent push. me and avital thought it best to not even mention "degraded" because it can be changed at any time.